packages/twenty-front/src/modules/favorites/components/FavoriteIcon.tsx

```diff
@@ -0,0 +1,35 @@
+import { ProcessedFavorite } from '@/favorites/utils/sortFavorites';
+import { useGetStandardObjectIcon } from '@/object-metadata/hooks/useGetStandardObjectIcon';
+import { useTheme } from '@emotion/react';
+import styled from '@emotion/styled';
+import { Avatar, useIcons } from 'twenty-ui';
+
+const StyledAvatar = styled(Avatar)`
+  background: inherit;
```

Comment:
same here, let's avoid style by the outside. And a grab icon on the parent might be enough, no need to apply it to the avatar?